pom.xml

```diff
@@ -77,7 +77,7 @@
     -->
     <javadoc.exec.goal>javadoc</javadoc.exec.goal>
 
-    <slf4jVersion>1.7.7</slf4jVersion>
+    <slf4jVersion>1.7.25</slf4jVersion>
```

Comment:
I'm with oleg here... If I use slf4j apis I can pick up newer than the core uses. 
This means core should supply a bom and we should not have this crazy mess here